### ISSUE TITLE
FR: Add Layers similar to photoshop

### ISSUE THREAD

**Issue by Kani3D** [zsviczian/obsidian-excalidraw-plugin#1386]
**Is your feature request related to a problem? Please describe.**
I would love to toggle between Layers to Hide/Show Information as needed.

**Describe the solution you'd like**
Overview of layers that can be toggled to show their specific content.

**Describe alternatives you've considered**
Using Visio or Photoshop would be my alternative.

**Additional context**
![image](https://github.com/zsviczian/obsidian-excalidraw-plugin/assets/105086349/f05f2519-9c10-4075-9e8c-ce35d2b3d6be)

**Comment by kloned** [comment]
It would be greatly beneficial, especially when presenting visually overwhelming content, to have the ability to switch layers on and off. This would aid in familiarizing oneself with the concept. For instance, a complex structure with numerous elements could be represented as major blocks on one layer, detailed components on a second layer, and the interconnections between elements on a third layer

**Comment by ThetaExpansion** [comment]
Yes, hoping this feature will be introduced, it should work well with hand-drawn elements

**Comment by SiliasGH** [comment]
Yeah, I also would be very interested in that feature!
